### PR TITLE
Don't detect D-M as D-M-M which causes multiseries header to be shown

### LINKED ISSUE #2784: Charts on dashboards are showing multiseries headers where they shouldn’t be
At least, that's how @tlrobinson described it. Here are some examples of what I'm seeing on our Stats instance. The words `count` and `open_issues` shouldn't be showing up by the card titles:

screen shot 2016-06-16 at 11 04 57 am

screen shot 2016-06-16 at 11 04 38 am

### PR DESCRIPTION
**Pull request by tlrobinson**
Resolves #2784

### CODE REVIEW

**Comment by tlrobinson** [comment]
@metabase/core-developers Can I get a review?

File: frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ const isDimensionDimensionMetric = (cols, strict = true) =>
     isMetric(cols[2])
 
 const isDimensionMetricMetric = (cols, strict = true) =>
```
Comment (salsakran) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx]: remind me of what "strict" means in this context?

Comment (tlrobinson) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx]: We use the same functions for determining "sensible" chart types and determining which type of multiseries chart (if any) to transform it to. `strict = true` causes each type to check that it has exactly the expected number and type of columns, and no more. The problem was D-M was being displayed as D-M-M, causing the multiseries header to be shown.

Comment (tlrobinson) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx]: That said, this is pretty confusing, and I'm not 100% sure it's correct any more...

Comment (agilliland) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx]: this looks right to me.  i mean, for this to qualify it should have 3 or more columns

Comment (tlrobinson) [inline frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx]: Yeah, that was my reasoning too, I'm just not sure if I'm passing the correct value for "strict" to `getChartTypeFromData` in every case. But I haven't noticed any other related bugs `¯\_(ツ)_/¯`

**Pull request merged**